src/main/java/techreborn/items/ManualItem.java

```diff
@@ -47,7 +47,15 @@ public ManualItem() {
 	@Override
 	public TypedActionResult<ItemStack> use(final World world, final PlayerEntity player, final Hand hand) {
 		if (player instanceof ServerPlayerEntity serverPlayerEntity) {
-			PatchouliAPI.get().openBookGUI(serverPlayerEntity, new Identifier("techreborn:techreborn"));
+            if (FabricLoader.getInstance().isModLoaded("patchouli")) { // TODO replace string by constant TechReborn.PATCHOULI_ID
+				PatchouliAPI.get().openBookGUI(serverPlayerEntity, new Identifier("techreborn:techreborn"));
+				// does 
+                // PatchouliAPI.get().openBookGUI(serverPlayerEntity, Registry.ITEM.getId(this));
+				// not work? Identifier techreborn:techreborn is a bad one btw
```

Comment:
It expects <modid>:<bookname> based on the patchouli reference guide.

I called the book techreborn (The name of the folder within the patchouli_books folder)